api/python/t4/api.py

```diff
@@ -449,3 +450,120 @@ def config(*autoconfig_url, **config_values):
         write_yaml(local_config, CONFIG_PATH, keep_backup=True)
 
     return HeliumConfig(CONFIG_PATH, local_config)
+
+def create_role(name, arn=None):
```

Comment:
I'm not really sure how much utility these methods add as written, they just wrap the single line `get_session()._REST_CALL_` . It seems like the response should be translating the response to a python object at least. And possibly operating on a python object in the case of edit (so`role_foo.edit_role(name,arn)`)